docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -508,8 +508,10 @@ this without having to do the math themselves.
 
     .. note::
 
-        When recovering prime factors this algorithm will always return ``p``
-        and ``q`` such that ``p < q``.
+        When recovering prime factors this algorithm will always returned ``p``
+        and ``q`` such that ``p > q``. Note: before 1.5, this function always
+        returned ``p`` and ``q`` such that ``p < q``. It was changed because
+        OpenSSL and related libraries generally require ``p > q``.
```

Comment:
OpenSSL doesn't require this though. How about `It was changed because libraries commonly require`?